src/main/java/com/powsybl/openloadflow/sensi/AcSensitivityAnalysis.java

```diff
@@ -40,104 +43,97 @@ public AcSensitivityAnalysis(MatrixFactory matrixFactory) {
         super(matrixFactory);
     }
 
-    private LfBranch getLfBranch(LfNetwork lfNetwork, String branchId) {
-        LfBranch branch = lfNetwork.getBranchById(branchId);
-        if (branch == null) {
-            throw new PowsyblException("Branch '" + branchId + "' not found");
+    protected void setBaseCaseSensitivityValues(List<SensitivityFactorGroup> factorGroups, DenseMatrix factorsState) {
```

Comment:
After reading in more detail your code, the function name is not the good one as you use it in base case sensitivity computation and in post contingency sensitivity computation.